ts/features/profile/saga/networking/handleGetProfile.ts

```diff
@@ -0,0 +1,46 @@
+import { ActionType } from "typesafe-actions";
+import { call, put } from "typed-redux-saga/macro";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { BackendClient } from "../../../../api/backend";
+import { SagaCallReturnType } from "../../../../types/utils";
+import { getProfile as loadProfile } from "../../store/actions";
+import { getGenericError, getNetworkError } from "../../../../utils/errors";
+
+/**
+ * Handle the remote call to retrieve the profile data
+ * @param getProfile
+ */
+export function* handleGetProfile(
+  getProfile: ReturnType<typeof BackendClient>["getProfile"],
+  _: ActionType<typeof loadProfile.request>
+) {
+  try {
+    const getProfileResult: SagaCallReturnType<typeof getProfile> = yield* call(
+      getProfile,
+      {}
+    );
+
+    if (getProfileResult.isRight()) {
+      // right side of the Either
+      if (getProfileResult.value.status === 200) {
+        yield* put(loadProfile.success(getProfileResult.value.value));
+        return;
+      } else {
+        yield* put(
+          loadProfile.failure({ ...getGenericError(new Error(`Error`)) })
```

Comment:
What do you think about put the status code in the error message?
This can be useful for monitoring purpose
```suggestion
          loadProfile.failure(getGenericError(new Error(postRegistraBeneficiarioResult.value.status.toString())) )
```

Reply:
What is `postRegistraBeneficiarioResult` 😄 ?

Reply:
Sorry, copy-paste! 🤦🏼‍♂️